src/common/notifications.cpp

```diff
@@ -37,7 +37,7 @@ namespace
     constexpr std::wstring_view PACKAGED_APPLICATION_ID = L"PowerToys";
     constexpr std::wstring_view APPIDS_REGISTRY = LR"(Software\Classes\AppUserModelId\)";
 
-    std::wstring APPLICATION_ID;
+    std::wstring APPLICATION_ID = L"Microsoft.PowerToysWin32";
```

Comment:
Do we have a common place to define non-localizable strings?

Reply:
this string is local to the file, and that's signified by placing it into anonymous namespace. just below this line we have `localized_strings` namespace, so it shouldn't be confusing.